### ISSUE TITLE
§ now acting as ` and the other way round - British keyboard

### ISSUE THREAD

**Issue by chriscrofty** [pqrs-org/Karabiner-Elements#3974]
Since updating to 15.20 these keys seem to have switched:

<img width="572" alt="image" src="https://github.com/user-attachments/assets/da802070-2f27-47e2-a6b4-8eaf113cde83">

Keyboards are set to ISO in virtual keyboard settings.

Macbook M3 pro, Apple with Magic Keyboard with Numeric Keypad 2.0.6. and same for internal keyboard.

**Comment by framilano** [comment]
Same exact thing here, Macbook M1 Pro, same exact keys have been inverted.

Turning off Karabiner fixes the issue

**Comment by wookiesh** [comment]
I have the same issue, just with a fr azerty layout, so inverted keys are (@,#) and (<,>).

**Comment by szduda** [comment]
Same issue on M1. I tried fighting fire with fire, ergo to fix that using karabiner itself, but I failed to find the § key on the lists.

**Comment by aplinxy9plin** [comment]
Hey, devs,  it's a big mistake most of us can't to work good now. Please fix it 🙏

**Comment by wookiesh** [comment]
Here's my temporary fix: add a complex modification:

```
{
    "description": "Temporary fix for inversed keys <> and @#",
    "manipulators": [
        {
            "from": {
                "key_code": "non_us_backslash",
                "modifiers": { "optional": ["shift"] }
            },
            "to": [{ "key_code": "grave_accent_and_tilde" }],
            "type": "basic"
        },
        {
            "from": {
                "key_code": "grave_accent_and_tilde",
                "modifiers": { "optional": ["shift"] }
            },
            "to": [
                {
                    "key_code": "non_us_backslash",
                    "modifiers": []
                }
            ],
            "type": "basic"
        }
    ]
}
```

I have not spotted an undesirable side effect yet.

**Comment by baglayan** [comment]
Can confirm, the same problem exists with the Turkish keyboard.

Can be worked around by mapping "grave_accent_and_tilde" to "non_us_backslash" and vice versa in Simple Modifications.

**Comment by Brawl345** [comment]
Same with German keyboard, < and ^ are swapped... so it's a general problem with these two keys (they are in the same place), not dependant on the language I guess.

**Comment by aplinxy9plin** [comment]
> Same with German keyboard, < and ^ are swapped... so it's a general problem with these two keys (they are in the same place), not dependant on the language I guess.

Yeah. It’s not dependent. I solved problem with change keyboard type from ISO -> US

**Comment by Brawl345** [comment]
>change keyboard type from ISO -> US

Worked for me, thanks!

**Comment by Wirone** [comment]
Swapping `~` (`grave_accent_and_tilde`) with `§` (`non_us_backslash`) was the main reason to use Karabiner for me to keep my habits, I even physically replaced these buttons to have desired layout. After upgrading to Sequoia and latest Karabiner it stopped working, and after spending some time on it (and landing here) it seems like all I had to do is... removing simple mappings, because keyboard works as expected without it 😅. I use ISO virtual keyboard.

**Comment by ccmjga** [comment]
My temporary fix
![image](https://github.com/user-attachments/assets/8358b765-5d25-4d10-97fe-9816e1a72290)
